test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -4204,6 +4204,31 @@ public async Task Restore_WithHttpsSourceIndexJsonReturningHttpResources_LogsNU1
             result.LockFile.LogMessages.Should().ContainSingle(m => m.Code == NuGetLogCode.NU1302 && m.Message.Contains("http://"));
         }
 
+        [Theory]
+        [InlineData("../")]
+        [InlineData("../my package")]
+        [InlineData("Some.Package/../newtonsoft.json")]
```

Comment:
Microsoft uses a bunch of fictional company names used in training materials and other demos.  Let's use one of them, like contoso or fabrikam, rather than a real package.